src/api.ts

```diff
@@ -7,311 +7,311 @@
  * Copy the `api.ts` source file into your project to use it.
  */ /** */
 
-import {DebugSession, Disposable, Event, Terminal} from 'vscode';
+import { DebugSession, Disposable, Event, Terminal } from 'vscode';
 
 /**
  * The result of executing a program.
  */
 export interface ExecutionResult {
-  /**
-   * The return code of the program.
-   */
-  retc: number|null;
-  /**
-   * The full standard output of the program. May be `` if standard out
-   * was not captured.
-   */
-  stdout: string;
-  /**
-   * Standard error output of the program. May be `` if standard error was
-   * not captured
-   */
-  stderr: string;
+    /**
+     * The return code of the program.
+     */
+    retc: number | null;
```

Comment:
It looks like it is also spacing around the union operator `|`. I think this one is ok though.